test/13_arraycommon.jl

```diff
@@ -0,0 +1,78 @@
+module TestArrayCommon
+
+using Base.Test
+using CategoricalArrays
+using NullableArrays
+using CategoricalArrays: DefaultRefType
+using Compat
+
+typealias String Compat.ASCIIString
+
+# == currently throws an error for Nullables
+(==) = isequal
+
+
+# Tests of vcat of CategoricalArray amd NullableCategoricalArray
+
+# Test that mergelevels handles mutually compatible ordering
+@test CategoricalArrays.mergelevels([6,3,4,7],[2,3,5,4],[2,4,8]) == ([6,2,3,5,4,7,8],true)
```

Comment:
In general, could you add spaces after commas? I'd rather keep a consistent style.